admin_guide/manage_scc.adoc

```diff
@@ -208,6 +195,77 @@ In order to preserve customized SCCs during upgrades, do not edit settings on
 the default SCCs other than priority, users, and groups.
 ====
 
+[[example-security-context-constraints]]
+=== Example Security Context Constraints Settings
+
+.Without Explicit runAsUser Setting
+----
+apiVersion: v1
+kind: Pod
+...
+fsGroup:
+  type:
+  ranges:
+  - min: 5000
+    max: 6000
+runAsUser: <1>
```

Comment:
Unfortunately, both examples won't work because Pod doesn't have such fields/values. Here is example of Kubernetes documentation about `SecurityContext` that could be used as illustration: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/